sdk/go/pulumi-language-go/language_test.go

```diff
@@ -238,6 +238,11 @@ func TestLanguage(t *testing.T) {
 						t.Skip("Skipping l1- tests in local mode")
 					}
 
+					// Skip provider tests for now, we test these with NodeJS and Python only.
```

Comment:
Do we have an associated issue here? If so, can we link it?

Reply:
No, I was pondering how to test this. Part of me thinks we should use the go-provider-framework but then it feels odd to be testing that in this repo.

Reply:
It would make sense for the go-provider-framework to live in this repo, but that's for another PR. In the mean time, I think a tracking issue is sufficient.